lib/route/link.c

```diff
@@ -945,6 +954,12 @@ static void link_dump_details(struct nl_object *obj, struct nl_dump_params *p)
 		nl_dump(p, "brd %s ", nl_addr2str(link->l_bcast, buf,
 						   sizeof(buf)));
 
+	if (link->ce_mask & LINK_ATTR_PERM_ADDR) {
+		uint8_t *addr = nl_data_get(link->perm_addr);
+		nl_dump(p, "hw %x:%x:%x:%x:%x:%x ", addr[0], addr[1], addr[2],
+			addr[3], addr[4], addr[5]);
```

Comment:
cannot blindly access the `addr` array. Must make sure that there are 6 elements there (and do something in case there isn't).